app/gateway/changelog.md

```diff
@@ -308,6 +308,22 @@ Kong Gateway version.
 * The function `kong.db.*.cache_key()` now falls back to `.id` if nothing from `cache_key` is found.
   [#8553](https://github.com/Kong/kong/pull/8553)
 
+### Known limitations
+
+* Kong Manager does not currently support the following features:
+  * Secrets management
+  * Plugin ordering
+  * Expression-based routing
+
+* Creating and updating Kong configuration for traditional mode is not allowed during migration from 2.x to 3.0.0.0.
+  * This is a known issue planned to be fixed in the next 2.8 release. If this is a requirement for upgrading, Kong operators should upgrade to that version before beginning a upgrade to 3.0.0.0.
+  * In traditional and hybrid mode Kong can be upgraded while continuing to proxy traffic.
+
+* Creating and updating Kong configuration for hybrid mode is not allowed from both a 2.8 CP and 3.0 CP.
```

Comment:
It is OK for 3.0 CP to push config to 2.8 DP for EE. We just don't recommend doing that.
And it should not be considered a "known limitation" to not allow different versions of CP to write to DB.